src/network/network.cpp

```diff
@@ -260,6 +261,43 @@ void NetworkGroup::Write(NetworkPacket& packet)
 	}
 }
 
+json_t * NetworkGroup::ToJson() const
+{
+	json_t * jsonGroup = json_object();
+	json_object_set_new(jsonGroup, "id", json_integer(id));
+	json_object_set_new(jsonGroup, "name", json_string(GetName().c_str()));
+	json_t * actionsArray = json_array();
+	for (size_t i = 0; i < actions_allowed.size(); i++) {
+		json_array_append_new(actionsArray, json_integer(actions_allowed[i]));
+	}
+	json_object_set_new(jsonGroup, "permissions", actionsArray);
+	return jsonGroup;
+}
```

Comment:
I think the serialisation of the permissions should be improved to something like:
```json
"permissions": [
    "chat",
    "terraform",
    "setWaterLevel",
    "clearLandscape"
]
```